docs/docs/faq.md

```diff
@@ -2,16 +2,23 @@
 Common questions which can come up during development.
 
 ## DLL not found Exception
-If the binaries are installed on the system but are not found during runtime a `DllNotFoundException` is raised. In this case the names of the installed libraries probably don't match the expected names.
 
-The gir.core nuget packages are build against 3 different package sources:
+The `System.DllNotFoundException` can be thrown in the following cases:
+
+- required DLLs or shared objects (the Unix equivalent of DLLs) are not installed in the operating system.
+- or `Module.Initialize()` was not called before instantiating an object in a namespace like `Gtk`, `Gdk` or `GtkSource`.
+- required DLLs or shared objects are installed, but still `DllNotFoundException` is thrown. In this case, the names of the installed libraries probably don't match the names expected by `gir.core`. This can happen when using a custom build binary.
```

Comment:
```suggestion
- required DLLs or shared objects are installed, but still `DllNotFoundException` is thrown. In this case, the names of the installed libraries probably don't match the names expected by `gir.core`. This can happen when [using a custom build binary](#how-can-i-use-gircore-with-a-custom-build-native-binary).
```